app/core/RPCMethods/RPCMethodMiddleware.ts

```diff
@@ -40,6 +40,7 @@ import { regex } from '../../../app/util/regex';
 import Logger from '../../../app/util/Logger';
 import DevLogger from '../SDKConnect/utils/DevLogger';
 import { addTransaction } from '../../util/transaction-controller';
+import Routes from '../../constants/navigation/Routes.js';
```

Comment:
This import is causing a build failure on Android. Removing the .js specifier fixes it.

```suggestion
import Routes from '../../constants/navigation/Routes';
```